src/Features/CSharp/Portable/ImplementInterface/AbstractChangeImplementationCodeRefactoringProvider.cs

```diff
@@ -71,8 +71,11 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
             // (like "GetEnumerator"), instead of the auto-generated names that the compiler makes
             // like: "System.IEnumerable.GetEnumerator"
             directlyImplementedMembers.AddRange(member, member.ExplicitOrImplicitInterfaceImplementations());
+            var implementations = member.ExplicitOrImplicitInterfaceImplementations();
+            if (implementations.IsEmpty)
+                return;
```

Comment:
Ideally, I think the refactoring should still be offered for this error scenario. But that scenario isn't very important so if that would be hard to support I think it's not a problem.

The confusing part here is that `ExplicitOrImplicitInterfaceImplementations` is empty, while `ExplicitInterfaceImplementations` is non empty:

https://github.com/dotnet/roslyn/blob/7eff5ed876f1d5f64dab9eb1b7d1c22e6ee587fa/src/Features/CSharp/Portable/ImplementInterface/CSharpImplementImplicitlyCodeRefactoringProvider.cs#L39-L41